### ISSUE TITLE
Found a typo in the README.md

### ISSUE THREAD

**Issue by mikkolukas** [lukasjapan/koreander#12]
`<script type="test/javascript">`
should probably be:
`<script type="text/javascript">`

\- or even better solution for output:
`<script>`
as MDN Web Docs writes:
> "The HTML5 specification urges authors to omit the attribute rather than provide a redundant MIME type."
> -- [developer.mozilla.org/en-US/docs/Web/HTML/Element/script](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script)

**Comment by lukasjapan** [comment]
Yes, I agree that the attribute should be omitted. Thanks for reporting.

Let me fix.
